src/cb_admin/pages/ForgotPassword.js

```diff
@@ -35,22 +36,15 @@ export default class ForgotPassword extends React.Component {
 
     CbAdmin.forgotPassword({ email: this.state.email })
       .then(() => this.props.history.push('/cb/login'))
-      .catch((error) => {
-        if (ErrorUtils.errorStatusEquals(error, 400)) {
-          this.setState({ errors: ErrorUtils.getValidationErrors(error) });
+      .catch((err) => {
+        if (ErrorUtils.errorStatusEquals(err, 400)) {
+          this.setState({ errors: ErrorUtils.getValidationErrors(err) });
 
-        } else if (ErrorUtils.errorStatusEquals(error, 401)) {
-          this.setState({ errors: { email: error.response.data.error } });
-
-        } else if (ErrorUtils.errorStatusEquals(error, 500)) {
-          this.props.history.push('/error/500');
-
-        } else if (ErrorUtils.errorStatusEquals(error, 404)) {
-          this.props.history.push('/error/404');
+        } else if (ErrorUtils.errorStatusEquals(err, 401)) {
+          this.setState({ errors: { email: err.response.data.error } });
```

Comment:
Is this right? In `Login.js` it's `err.response.data.error.message`.

Reply:
good spot, error response was mocked incorrectly in the tests too.

Fixed in amend.